docs/docs/api/nativeMethods/measure.md

```diff
@@ -46,13 +46,11 @@ const Comp = () => {
 
   useDerivedValue(() => {
     const measured = measure(aref);
-    // ...
+    if (measured !== null) {
+      // ...
```

Comment:
Maybe let's add some `console.log` here that makes use of the props of `measured`.